crawl-ref/source/religion.cc

```diff
@@ -3905,11 +3905,26 @@ bool god_likes_spell(spell_type spell, god_type god)
     }
 }
 
+/**
+ * Does your god hate spellcasting?
+ *
+ * @param god           The god to check against
+ * @return              Whether the god hates spellcasting
+ */
 bool god_hates_spellcasting(god_type god)
 {
     return god == GOD_TROG;
 }
 
+/**
+ * Will your god put you under penance if you actually cast spell?
+ *
+ * @param spell         The spell to check against
+ * @param god           The god to check against
+ * @param fake_spell    Whether the spell is evoked, or from an innate or
```

Comment:
good idea. I just copied it from another docstring in spl-cast.cc. 

Looks like spl-cast.cc, religion.cc, and spl-util.cc all have different wording for this param. Maybe I'll just change them all to the same thing (and see if I can find anymore examples lurking about)